crates/holochain_types/src/app.rs

```diff
@@ -548,8 +631,9 @@ impl InstalledAppCommon {
                 let role = AppRoleAssignment {
                     base_cell_id: cell_id,
                     is_provisioned: true,
-                    clones: HashSet::new(),
-                    clone_limit: 0,
+                    clones: HashMap::new(),
+                    clone_limit: 256,
```

Comment:
why this instead of e.g. u8::MAX (which is 255)? does this limit have anything to do with u8 or is that a coincidence?

Reply:
It's an arbitrary limit right now, I might as well use u8::MAX.